docs/openapi/api.yaml

```diff
@@ -77,6 +79,10 @@ paths:
     $ref: './audits-api.yaml#/audits-for-site'
   /sites/{siteId}/latest-audit/{auditType}:
     $ref: './audit-api.yaml#/latest-audit-for-site'
+  /slack/create:
```

Comment:
i would use a more detailed name like `createAEMChannelWithSuffix` - possibly in the future we'd have more creates, i.e. `createChannelWithName` or `createCanvas` just as an example. the implementation seems to mandate `#aem-` as the channel prefix, so it is not generic enough for `createChannel`?

Reply:
I agree `create` is too generic. The detailed naming should align with the existing dash-case used in paths.
What about `POST /slack/channels` to create a channel?

Reply:
i was thinking about this too, but then rejected the idea as it smelled like a resource-centric API, which it is definitely not. slack channels are not known resources/entites in our system. the hooks are command-style interfaces, not resource-style.

Reply:
This is not in the hooks group. And it says: "Slack channel management" which to me sounds like a management API, hence the proposal.

Reply:
ok, my mistake, it's not hooks, but also not a resource api